### ISSUE TITLE
Desktop sharing with desktop sound

### ISSUE THREAD

**Issue by alainvanderbroeck** [peer-calls/peer-calls#127]
Hi,

When i play a video, the other ones cannot hear the video audio.
I think your code does not support hat yet.
Is this something you have planned for the future?

**Comment by jeremija** [comment]
Yeah currently no audio is being sent. This is to prevent feedback. For example it might transfer back the audio of all speakers and thus cause feedback.

I'm wondering what's your use case for sending computer audio?

**Comment by alainvanderbroeck** [comment]
Thanks for you answer.

I would like to preview a video production with a customer. (without uploading it to the cloud, because it is not finished yet)

**Comment by jeremija** [comment]
Interesting! I'm going to try to add an option to share the computer audio as well. I'm just kind of swamped with other projects at the moment so it might take a few weeks.

**Comment by alainvanderbroeck** [comment]
Yes, we miss that in other applications like teamviewer. That is why we tried your solutions. Works well!
When you add this thing, you would make your app unique (in my opinion).

**Comment by alainvanderbroeck** [comment]
Hi jeremija. 
Did you already have time to look into this feature request?

**Comment by jeremija** [comment]
Hi @alainvanderbroeck, I just pushed version 4.0.9 which has support for this. I'm waiting for the build to complete and then I'm going to push it to peercalls.com.

<img width="191" alt="image" src="https://user-images.githubusercontent.com/1489493/91635680-a5bdda00-e9fa-11ea-909f-05bf645c3cb8.png">

However, I noticed that in Chrome it only works when choosing a Chrome tab to share (and you have to select the `Share audio` checkbox at the bottom of the picker provided by Chrome UI) - it doesn't work for sharing the whole desktop. This is  a browser restriction, and there doesn't seem to be anything I can do about it.

I couldn't get it to work in Firefox and I haven't tried Safari yet. Please let me know what your experience is.

**Comment by alainvanderbroeck** [comment]
Thank you for your quick response.
We are looking forward to this feature. Many thanks!

We are going to try it this week. We'll let you know what our experience is.

**Comment by jeremija** [comment]
Great - I'd recommend you to take a look at the Browser Compatibility table here: https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getDisplayMedia#Browser_compatibility

Looks like Chrome and Edge on Windows and Chrome OS/Edge OS can capture audio of the whole machine, whereas on Mac and Linux it's limited to selecting a single Chrome tab only.
